mmdet/datasets/xml_style.py

```diff
@@ -44,6 +75,8 @@ def get_ann_info(self, idx):
         labels_ignore = []
         for obj in root.findall('object'):
             name = obj.find('name').text
+            if name not in self.CLASSES:
+                continue
```

Comment:
Yes. Because sometimes the annotation of an image contains instances of not only customized classes but also other classes. Those unused classes should be marked as background.